oauth2_http/java/com/google/auth/oauth2/IdentityPoolCredentials.java

```diff
@@ -269,22 +276,24 @@ private String getSubjectTokenFromMetadataServer() throws IOException {
     }
   }
 
+  /**
+   * Returns whether or not the current configuration is for Workforce Pools (which enable 3p user
+   * identities, rather than workloads).
+   */
+  private boolean isWorkforcePoolConfiguration() {
```

Comment:
Should be public so developers can easily distinguish between two configurations.